radio/src/translations/en.h.txt

```diff
@@ -1020,7 +1020,7 @@
 #define TR_SET                         BUTTON("Set")
 #define TR_TRAINER                     "Trainer port"
 #define TR_ANTENNAPROBLEM              CENTER "TX antenna problem!"
-#define TR_MODELIDUSED                 TR("ID already used","Model ID already used")
+#define TR_MODELIDUSED                 TR("ID used in :","Receiver ID used in :")
```

Comment:
in english there is a space after the colon, not before
if I am not mistaken of course